src/modules/back-to-top/back-to-top.component.spec.ts

```diff
@@ -24,7 +24,7 @@ describe('StacheBackToTopComponent', () => {
     debugElement = fixture.debugElement;
   });
 
-  beforeEach(inject([WindowRefService], service => {
+  beforeEach(inject([WindowRefService], (service: any) => {
```

Comment:
Let's pull this out into a hotfix branch (or make @blackbaud-brandonhare fix it).